docs/src/literate-tutorials/maxwell.jl

```diff
@@ -0,0 +1,72 @@
+# Maxwell's equations 
+# For simplicity, we start with the very basic example from 
+# https://www.math.colostate.edu/~bangerth/videos.676.33.html
+# Specifically, 
```

Comment:
Maybe some context might help, because there is quite a big jump from Maxwell to this specific form
```suggestion
# Specifically, if we investigate an electromagentic problem where we are interested in the steady state of either the electrical or magentical field, in the case when neither electrical nor magnetical sources are present, then we can state the weak form as follows
```

Reply:
Maybe we can also leave the hint for users that we can simplify further to obtain a Poisson or Laplace problem instead for this very specific case and that we only look at this case here to showcase how to use Nedelec elements to assemble a Hcurl conforming problem.